app/assets/javascripts/components/about_page.es6.jsx

```diff
@@ -0,0 +1,131 @@
+                </h5>
+                <p className="section__text">
+                  Forte's roots trace back to 2008, when our CEO formed a service organization at Homestead<br></br>
+                  High School allowing student musicians to collaborate via small ensembles. Ranging from<br></br>
+                  duos to quintets, these intimate groups inspired levels of student independence and<br></br>
+                  creativity greater than those typically afforded by school-wide bands, orchestras or choirs.<br></br>
+                  Through this organization, students connected with and enriched the local community via<br></br>
+                  public performances for audiences including HCR ManorCare, the Santa Clara Valley Blind<br></br>
+                  Center, and the J.W. House of Kaiser Permanente.
+                </p>
+                <p className="section__text">
+                  In this spirit, we launched Forte in 2015 with the vision of leveraging music education as a<br></br>
+                  platform to enrich the lives of the youth who otherwise may not have the resources or guidance<br></br>
+                  to pursue their musical passions.
+                </p>
+              </div>
+              <div className="section">
+                <h2 className="section__title">
+                  Who are we?
+                </h2>
+                <div className="section__headshots-container">
+                  <div className="section__individual-container">
+                    <div>
+                      <img className="section__image" src={ImageConstants.headshots.daniel} href="#" />
+                    </div>
+                    <div className="section__person-info">
+                      <div className="section__name">
+                        Daniel Kim
+                      </div>
+                      <div className="section__position">
+                        Chief Executive Officer
+                      </div>
+                    </div>
+                  </div>
+                  <div className="section__individual-container">
+                    <div>
+                      <img className="section__image" src={ImageConstants.headshots.richard} href="#" />
+                    </div>
+                    <div className="section__person-info">
+                      <div className="section__name">
+                        Richard Donahue
+                      </div>
+                      <div className="section__position">
+                        Chief Financial Officer
+                      </div>
+                    </div>
+                  </div>
+                  <div className="section__individual-container">
+                    <div>
+                      <img className="section__image" src={ImageConstants.headshots.shaine} href="#" />
+                    </div>
+                    <div className="section__person-info">
```

Comment:
So, for all of these section__person-info, I think you can abstract it into a function called renderIndividual! Take a look at shimmy's pull request to see how he did it :) Let me know if you need more examples/info

I think it will clean up the code a bit and make the html shorter :)